client/wildcard/src/components/Form/internal/BaseControlInput.module.scss

```
@@ -1,3 +1,3 @@
 .label:hover {
-    color: var(--gray-12);
+    color: var(--input-hover-color);
```

```suggestion
    color: var(--input-label-hover-color);
```

Nit: Let's specify this just for labels so it's not confusing if we ever add a hover style for the actual inputs